delete.go

```diff
@@ -56,8 +62,18 @@ status of "ubuntu01" as "stopped" the following will delete resources held for
 				}
 			}
 			return fmt.Errorf("container init still running")
+		case libcontainer.Running:
+			force := context.Bool("force")
+			if force {
+				if err := container.Signal(syscall.SIGKILL); err != nil {
+					return fmt.Errorf("Could not stop running container, cannot remove - %v", err)
+				}
+				destroy(container)
```

Comment:
The reason why the spec does not require “delete a running container” to work is that “I sent it a kill signal” doesn't mean that it's dead (it could be in an uninterruptible sleep disk sleep, see proc(5)), and the delete process has no way to block on the container process's actual exit.  See [here][1].

[1]: https://github.com/opencontainers/runtime-spec/pull/384#discussion_r64801501